packages/components/src/tooltip/index.js

```diff
@@ -24,7 +24,65 @@ import Shortcut from '../shortcut';
  *
  * @type {number}
  */
-const TOOLTIP_DELAY = 700;
+export const TOOLTIP_DELAY = 700;
+
+const eventCather = (
+	<div
+		className="event-catcher"
+		style={ {
+			zIndex: 9999,
+			position: 'absolute',
+			width: '100%',
+			height: '100%',
+			top: 0,
+		} }
+	/>
+);
+
+const getDisabledElement = ( { eventHandlers, child, childrenWithPopover } ) =>
+	cloneElement(
+		<span style={ { position: 'relative' } }>
```

Comment:
Same comment with inline style as above. I may have missed this kind of styling, is it also used in other components?